chain/client/src/client_actor.rs

```diff
@@ -1757,13 +1757,16 @@ impl ClientActor {
             None
         };
 
-        let epoch_identifier = ValidatorInfoIdentifier::BlockHash(head.last_block_hash);
-        let validator_epoch_stats = self
-            .client
-            .runtime_adapter
-            .get_validator_info(epoch_identifier)
-            .map(get_validator_epoch_stats)
-            .unwrap_or_default();
+        let validator_epoch_stats = if is_syncing {
+            Default::default()
```

Comment:
This may be confusing for someone who looks at the stats. Could we display it in a better way?